ddtrace/llmobs/_integrations/openai.py

```diff
@@ -187,10 +187,10 @@ def _llmobs_set_meta_tags_from_chat(
                 continue
             input_messages.append({"content": str(getattr(m, "content", "")), "role": str(getattr(m, "role", ""))})
         span.set_tag_str(INPUT_MESSAGES, json.dumps(input_messages))
-        parameters = {"temperature": kwargs.get("temperature", 0)}
-        if kwargs.get("max_tokens"):
-            parameters["max_tokens"] = kwargs.get("max_tokens")
+
+        parameters = {k: v for k, v in kwargs.items() if k not in ("model", "messages", "tools", "functions")}
```

Comment:
is there a guarantee that these parameters are json serializable? If not might be worth to add a try/catch here, unless this method call is being wrapped in another try/catch

Reply:
Good catch, will add a default repr if the object is not json serializable.